apps/next-evans/lib/db/friends.ts

```diff
@@ -104,19 +170,28 @@ async function getFriendsFromDB(lang: Lang): Promise<Record<string, FriendType>>
           created_at: true,
           slug: true,
           partial_description: true,
+          featured_description: true,
           id: true,
+          alt_language_id: true,
```

Comment:
i see you added a bunch of `featuredDescription`s to all of the `Omit`s, but does this line mean we should be omitting `altLanguageId` as well? it's this sort of thing that makes me wonder if we should go `Pick` only.